v_6_0_0/files/k8s-resource/calico-all.yaml

```diff
@@ -286,21 +286,23 @@ metadata:
   name: calico-node
   namespace: kube-system
   labels:
+    app.kubernetes.io/name: calico-node
```

Comment:
Please make sure this is communicated. Peoples shell history will stop working.

Reply:
It will keep working as I did not remove the old label :) I will communicate it though